srcpkgs/flatbuffers/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'flatbuffers'
+pkgname=flatbuffers
+version=1.10.0
+revision=1
+build_style=cmake
+short_desc="Memory Efficient Serialization Library"
+maintainer="Elie ROUDNINSKI <xademax@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://google.github.io/flatbuffers/"
+distfiles="https://github.com/google/${pkgname}/archive/v${version}.tar.gz"
+checksum=3714e3db8c51e43028e10ad7adffb9a36fc4aa5b1a363c2d0c4303dd1be59a7c
+
+if [ "${CROSS_BUILD}" ]; then
```

Comment:
I don't think using curly braces here is idomatic for void

Reply:
If you really mind the curly braces around cross_build, i can remove them, even though, i prefer when all variables are curly. Having rules to say in which context var should by curly braced is cumbersome imo

Reply:
I just use template-formalize, which automatically deals with stuff like that: https://github.com/maxice8/meltryllis/blob/master/bin/template-formalize#L162

Reply:
I am not that stubborn, so if you think its better, then ill do it right away.